cli/bench_command.go

```diff
@@ -107,6 +109,8 @@ func configureBenchCommand(app commandHost) {
 	addPubFlags := func(f *fisk.CmdClause) {
 		f.Flag("multisubject", "Multi-subject mode, each message is published on a subject that includes the publisher's message sequence number as a token").UnNegatableBoolVar(&c.multiSubject)
 		f.Flag("multisubjectmax", "The maximum number of subjects to use in multi-subject mode (0 means no max)").Default("100000").IntVar(&c.multiSubjectMax)
+		f.Flag("payload", "File containing a message payload to send").StringVar(&c.payloadFilename)
```

Comment:
using `ExistingFileVar()` instead of `StringVar()` and Fisk will handle error handling for non existing files for you

Reply:
Didn't know about ExistingFile/Dir, neat!